src/interfaces/vault.ts

```diff
@@ -106,6 +114,9 @@ export class VaultInterface<T extends ChainId> extends ServiceInterface<T> {
         earnings => earnings.assetAddress === asset.address
       )?.dayData;
 
+      // workaround for api.yearn.finance returning different attributes for backscracher vaults
+      dynamic.metadata.apy = oldApyToSnakeCase(dynamic.metadata.apy);
```

Comment:
I think it'd be better to put this logic in the `apy` function in `src/services/vision.ts` so that anyone that wants to consume the apy data from the sdk will get the correct values, and not just for `vaults.get()`